src/redux/reducers/democracy.js

```diff
@@ -15,6 +15,19 @@ const initialState = {
 
 const democracyReducer = handleActions(
   {
+    [combineActions(
+      democracyActions.voteForCongress.call,
+    )]: (state) => ({
+      ...state,
+      loading: true,
+    }),
+    [combineActions(
+      democracyActions.voteForCongress.success,
+      democracyActions.voteForCongress.failure,
+    )]: (state) => ({
+      ...state,
+      loading: false,
```

Comment:
Why not use the `gettingDemocracyInfo` that's already used by all other democracy actions for this purpose?